incydr/cli/cmds/file_events.py

```diff
@@ -160,12 +162,19 @@ def yield_all_events(q: EventQuery):
         render.csv(FileEventV2, events, columns=columns, flat=True)
     elif format_ == TableFormat.table:
         render.table(FileEventV2, events, columns=columns, flat=False)
-    elif format_ == TableFormat.json:
-        for event in events:
-            console.print_json(data=event)
-    else:  # format_ == "raw-json"/TableFormat.raw_json
-        for event in events:
-            console.print(json.dumps(event), highlight=False)
+    else:
+        printed = False
+        if format_ == TableFormat.json:
+            for event in events:
+                printed = True
+                console.print_json(data=event)
+        else:  # raw-json
+            for event in events:
+                printed = True
+                event = json.dumps(event)
+                click.echo(event)
```

Comment:
found out using `console.print` here would either wrap the line to the size of the console (which would add breaking newlines if you piped to a file), so we should stick to `click.echo` for raw-json everywhere to make sure it's always on one line.